solarforecastarbiter/conftest.py

```diff
@@ -83,3 +83,26 @@ def ac_power_forecast_metadata(site_metadata):
         site=site_metadata
     )
     return ac_power_fx_meta
+
+
+def fixed_modeling_parameters():
+    modeling_params = datamodel.FixedTiltModelingParameters(
+        ac_capacity=.003, dc_capacity=.0035, temperature_coefficient=-0.003,
+        dc_loss_factor=3, ac_loss_factor=0,
+        surface_tilt=30, surface_azimuth=180)
+    return modeling_params
+
+
+def tracking_modeling_parameters():
+    modeling_params = datamodel.SingleAxisModelingParameters(
+        ac_capacity=.003, dc_capacity=.0035, temperature_coefficient=-0.003,
+        dc_loss_factor=3, ac_loss_factor=0,
+        axis_tilt=0, axis_azimuth=0, ground_coverage_ratio=2/7,
+        backtrack=True, maximum_rotation_angle=45)
+    return modeling_params
+
+
+@pytest.fixture(scope='module',
+                params=['fixed', 'tracking'])
+def modeling_parameters(request):
```

Comment:
I think this could return a tuple of (modeling_parameters, system_type) which you can unpack in the test function instead of the current exception handling. I not sure of a nicer way to do it

Reply:
to be clear... where system_type = 'fixed' or 'tracking' strings?

Reply:
yes